tests/foreman/ui/test_contentview.py

```diff
@@ -3591,8 +3591,9 @@ def test_positive_composite_child_inc_update(session):
             errata = version['errata']['table']
             assert len(errata) > 1
             assert (FAKE_0_INC_UPD_ERRATA in {row['Errata ID'] for row in errata})
-            packages = version['rpm_packages']['table']
-            assert len(packages) > 1
+            packages = session.contentview.search_version_package(composite_cv.name,
+                                                                  expected_version,
+                                                                  'name=pulp-test-package')
```

Comment:
Could we derive `name` value from `FAKE_0_INC_UPD_NEW_PACKAGE` somehow? So if we ever change the constant, we don't have to update value hardcoded here?
Below works for me, but you might use some more readable way:
```python
"-".join(FAKE_0_INC_UPD_NEW_PACKAGE.split('-')[0:-2])
```

Reply:
`productmd` contains a function to parse NVRAs, perhaps we should use it here. I used it in the Compose testing tool. 

https://github.com/release-engineering/productmd/blob/6e70821f53714f806b7ffe895481bd0b7ecb3f62/productmd/common.py#L79-L95

`productmd` is published by Red Hat release engineering and basically has a bunch of functions to parse their metadata. Looks like we don't seem to include it in robottellos requirements, but I think we should included it since it has a lot of useful stuff. IMHO its probably can be considered the definitive source of truth for things like this. 

If other agree Lai can you add productmd as a dependency in robotello, and use `parse_nvra` as part of this.

Reply:
@latran feel free to add this dependency as part of this PR.

Reply:
@pgagne @mirekdlugosz 
Pulled in nvra dependency and used it for your suggested improvement.  Retested and everything works well.